src/styles/blocks/about-us.scss

```diff
@@ -0,0 +1,31 @@
+.about-us {
+  @include pageGrid;
+
+  &__title {
+    grid-column: 1 / -1;
+
+    @include onTablet {
+      grid-column: 1 / 4;
+    }
+
+    @include onDesktop {
+      grid-column: span 6;
+    }
+  }
+
+  &__text {
+    grid-column: 1 / -1;
+
+    font-family: 'Space Mono', monospace;
+    font-size: 16px;
```

Comment:
Consider defining font properties in a centralized location (main.scss or _typography.scss) and using variables or mixins for consistency.